DQM/Physics/python/DQMPhysics_cff.py

```diff
@@ -33,6 +34,7 @@
                            *HiggsDQM
                            *ExoticaDQM
                            *B2GDQM
+                           *CentralitypADQM
```

Comment:
@slava77 Can you give us a hint on how to append the centrality DQM only for the case when the pA era is used?  Thanks!